### PR TITLE
make bootstrap work on machines where python is python3

### PR DESCRIPTION
**Pull request by acharis**
when i was getting started this really tripped me up a bunch.
i'm on arch linux, which might not be a big deal. but freebsd also has python3 as default.

we can debate this. i don't feel super strongly about it, just wanted to get it out there.

### CODE REVIEW

**Comment by acharis** [comment]
are you not in favor of this change in principle? or would you be alright with it if everything worked on, say, ubuntu (and travis of course)?

i suppose there are other ways to get compatibility on systems where python means python3, but i think we're a long way from getting this working on systems that don't have python2 installed at all.

**Comment by michael-berlin** [comment]
I agree that the current code is probably not compatible with Python 3 and that you'll always have to have Python 2.7 installed. I think that's acceptable. If somebody wants to make the code compatible with Python 3, be my guest! :)

Regarding those users where the standard Python binaries point to Python 3 and not 2:

I did a quick search on the Internet and it looks like Python 3 binaries usually have the "3" suffix and not the other way around.

Even the upcoming Ubuntu provides /usr/bin/python as Python 2.7 and not version 3:

http://packages.ubuntu.com/search?suite=zesty&arch=any&searchon=contents&keywords=%2Fusr%2Fbin%2Fpython

http://askubuntu.com/a/475815 is also an interesting read. It looks like Ubuntu may change this eventually as well.


If we want our scripts to work on both "python3" and "python2" systems, we'll probably have to introduce a variable for each binary and set it depending on the system. That's going to look very ugly, but I would be okay with that. Can you think of another solution?

**Comment by acharis** [comment]
I'm definitely not volunteering to make the python in here compatible with v2+v3! I agree that it's acceptable to require python2.7 (at least until 2020). PEP 394 is recommending distros to have a python2 symlink, but ubuntu doesn't do this, so we're stuck with the kind of ugliness that is this second commit.

**Comment by acharis** [comment]
current failure is the intermittent thing we've been seeing

File: dev.env [hunk 1]
```diff
@@ -35,6 +35,13 @@ export PYTHONPATH=$(prepend_path $PYTHONPATH $VTROOT/py-vtdb)
 export PYTHONPATH=$(prepend_path $PYTHONPATH $VTTOP/test)
 export PYTHONPATH=$(prepend_path $PYTHONPATH $VTTOP/test/cluster/sandbox)
 
+command -v python2 >/dev/null 2>&1 && PYTHON=python2 || PYTHON=python
```
Comment (michael-berlin) [inline dev.env]: Please add a comment before this block to clarify why we're doing this.

Comment (acharis) [inline dev.env]: for sure

**Pull request merged**